### ISSUE TITLE
XSS issue. Unable to escape alt text.

### ISSUE THREAD

**Issue by neverthemachineforever** [miromannino/Justified-Gallery#47]
I'm having an issue where I am binding html to the alt attributes of images which are inside of a gallery.

Especially when used in combination with something like Swipebox, it's nice to allow HTML here so that image captions can contain formatting such as italics. This does however have the side effect of making the (hover) alt text show markup.

Is having HTML within the alt attribute a valid use of this library?

My case is an output issue and sanitising the data server-side is not an option.

This is demonstrated here:
http://jsfiddle.net/p0fm2rp6/1/

Workarounds welcome.
(awesome plugin by the way :smile: )

**Comment by seanbeaton** [comment]
Just as a note to why this addition is important - sometimes we we're allowing user content in the alt attribute, making this an XSS risk. 

As of now, the default setting keeps the current functionality (no filtering), but it may be worth looking at having the default be safer and use the `true` or `'lenient'` option to guard against accidental XSS vulnerabilities. This would change functionality for people intentionally using JS in their captions (for like buttons and such), so it would need to be a separate version with documentation of the change for anyone upgrading.
